src/prebid.js

```diff
@@ -156,9 +156,10 @@ $$PREBID_GLOBAL$$.getBidResponsesForAdUnitCode = function (adUnitCode) {
 /**
  * Set query string targeting on one or more GPT ad units.
  * @param {(string|string[])} adUnit a single `adUnit.code` or multiple.
+ * @param {function(object)} customSlotMatching gets a GoogleTag slot and returns a filter function.
```

Comment:
Can you update the description and add a usage example here?
ie, this is a a filter function that if passed will override the default matching for a slot. 
``` customSlotMatching(slot) = > slot.getAdUnitPath() === 'someIdPassed' ```